apertium-uzb.uzb.lexc

```diff
@@ -90,17 +126,18 @@ Multichar_Symbols
 %<title%>                         !
 
 !! Tense / finite forms
-%<pres%>      ! for "жатыр", "тұр", "отыр" and "жүр"
+%<pres%>      ! for "yotir", "dir",..
 %<aor%>       ! -{E}
-%<past%>      ! -{G}{A}н
+%<past%>      ! -{G}{A}n
 %<ifi%>       ! -{D}{I}
-%<evid%>      ! -{I}пт{I}
+%<evid%>      ! -{I}pt{I}
```

Comment:
`<ifi><evid>` is `-{i}btir` in Uzbek, I think.  `<past><evid>` should be `-gan ekan`.

Reply:
Noted. It must be correct now.